app/styles/organisms/wizard-container.less

```diff
@@ -1,6 +1,4 @@
 .wizard-container {
-  flex: 1;
```

Comment:
For any reviewer questioning this, it is already defined below, just hidden by github by default !